### PR TITLE
Feature/Fix E2E error on Header languages 

### PR DESCRIPTION
**Pull request by adorapham**
Fix the error on verifying languages when running the E2E failing on staging

### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

**Comment by adorapham** [comment]
hi @galeza @jessica-lau , when i run the E2E and see header failing 24 scenarios with error - failed because this element is detached from the DOM.  Would you please check if you see the same issue.  I also make the fix for this issue in this PR#2128 and please see if you have any suggestion to make it simpler.

**Review by TimeaKovacs86 — DISMISSED** [review]
Looks good to me. Thank you @adorapham  
Fix E2E header languages verification passed

File: cypresse2e/cypress/integration/e2eTests/header/header.spec.js [hunk 1]
```diff
@@ -308,7 +308,13 @@ describe("FightPandemics Navigation bar", () => {
     //after clicking style is changed to bold
     language
       .click({ force: true })
-      .should("have.attr", "style", "font-weight: bold;");
+      .get(languageSelector)
+      .should(($languageEl) => {
+        expect($languageEl)
+          .to.have.attr("style")
+          .contains("font-weight: bold;");
+      });
```
Comment (jessica-lau) [inline cypresse2e/cypress/integration/e2eTests/header/header.spec.js]: Here's another way to write the code with less text but your way works too

```
language
      .click({ force: true })
      .get(languageSelector)
      .should("have.attr", "style", "font-weight: bold;");
```
image

**Review by jessica-lau — DISMISSED** [review]
LGTM. See below for another option to write the code :)

File: cypresse2e/cypress/integration/e2eTests/header/header.spec.js (hunk 1 above)
Comment (adorapham) [reply cypresse2e/cypress/integration/e2eTests/header/header.spec.js]: TY @jessica-lau

**Review by galeza — COMMENTED** [review]
@adorapham 
Which version will you decide to keep? Yours or simplified, proposed by @jessica-lau ?

**Comment by adorapham** [comment]
> @adorapham
> Which version will you decide to keep? Yours or simplified, proposed by @jessica-lau ?

@galeza i decide to change as Jessica 's suggestion
image

File: cypresse2e/cypress/integration/e2eTests/header/header.spec.js (hunk 1 above)
Comment (adorapham) [reply cypresse2e/cypress/integration/e2eTests/header/header.spec.js]: I decided to fix it as recommened

**Review by TimeaKovacs86 — APPROVED** [review]
Tests pass! TY

**Review by galeza — APPROVED** [review]
Lgtm

**Pull request merged**